Extensions/quick_tags.js

```diff
@@ -398,20 +400,26 @@ XKit.extensions.quick_tags = new Object({
 	},
 
 	do_posts: function() {
+		(async () => {
```

Comment:
does this need to be an IIFE? is there a problem with making `do_posts` an async function?